libraries/AP_Camera/AP_Camera.h

```diff
@@ -46,11 +47,28 @@ class AP_Camera {
     void            control(float session, float zoom_pos, float zoom_step, float focus_lock, float shooting_cmd, float cmd_id);
 
     // set camera trigger distance in a mission
-    void            set_trigger_distance(uint32_t distance_m)
+    void            set_trigger_distance(float distance_m)
     {
         _trigg_dist.set(distance_m);
     }
 
+    // CAMPOS methods
+    void            set_campos_parameters(uint8_t positions, float roll, float pitch, int16_t interval)
+    {
+        _campos_positions = positions;
+        _campos_roll.set(roll);
+        _campos_pitch.set(pitch);
+        _campos_angle_interval.set(_campos_roll * 2 / (_campos_positions - 1));
+        _campos_pose_counter = 0;
+        _campos_updated_roll_angle = false;
+
+        if (interval > 0)
```

Comment:
the curly bracket should be on the same line as the "if"

Reply:
I will update that.